akka-sample-cqrs-scala/src/main/scala/sample/cqrs/Main.scala

```diff
@@ -20,6 +21,7 @@ import akka.projection.eventsourced.scaladsl.EventSourcedProvider
 import akka.stream.alpakka.cassandra.scaladsl.CassandraSessionRegistry
 import com.typesafe.config.Config
 import com.typesafe.config.ConfigFactory
+import sample.cqrs.grpc.ShoppingCartGrpcServer
 
 object Main {
```

Comment:
To run this sample you can start the write side as described in README, then start the other service with:
```
sbt "runMain sample.cqrs.otherservice.Main 3551"
```

update with curl and you will see the events propagated to otherservice.